enterprise/audit/table.go

```diff
@@ -275,6 +275,7 @@ var auditableResourcesTypes = map[any]map[string]Action{
 		"id":           ActionIgnore,
 		"name":         ActionTrack,
 		"description":  ActionTrack,
+		"deleted":      ActionIgnore,
```

Comment:
We want this to show up in the audit log diffs 
```suggestion
		"deleted":      ActionTrack,
```